clients/drcachesim/tools/opcode_mix.cpp

```diff
@@ -79,8 +79,10 @@ bool
 opcode_mix_t::process_memref(const memref_t &memref)
 {
     if (!type_is_instr(memref.instr.type) &&
-        memref.data.type != TRACE_TYPE_INSTR_NO_FETCH)
-        return "";
+        memref.data.type != TRACE_TYPE_INSTR_NO_FETCH) {
+        error_string = "";
```

Comment:
This only needs to be set when returning false

Reply:
Done.